Formula/trinity.rb

```diff
@@ -35,6 +37,23 @@ class Trinity < Formula
   fails_with :clang # needs openmp
 
   def install
+    ENV.cxx11
```

Comment:
I thought this line was a no-op? I may be wrong. I'm probably wrong.

Reply:
https://github.com/Homebrew/brew/blob/1644ec2f50047f026781486f37a9c26c2a64bf90/Library/Homebrew/extend/ENV/super.rb#L271-L274
So it's not a no-op, but I still thought that C++11 was the default standard for GCC 5 and modern Clang.